regexp-diff.md

```diff
@@ -0,0 +1,31 @@
+# RegExp changes
```

Comment:
This file should be removed from the PR now, right?

Reply:
yep